Sources/FluentKit/Properties/Parent.swift

```diff
@@ -8,7 +8,7 @@ public final class Parent<To>
     public var wrappedValue: To {
         get {
             guard let value = self.value else {
-                fatalError("Parent relation not eager loaded, use $ prefix to access")
+                fatalError("Parent relation \(name) not eager loaded, use $ prefix to access")
```

Comment:
This should be added for all of the relations for consistency. The name might also be a bit long, so it could be nice to format this like:

```swift
fatalError("Parent relation not eager loaded, use $ prefix to access: \(self.name)")
```

Reply:
Thanks @tanner0101 for the input—tweaked the messaging slightly for the other relations to make them consistent throughout. LMK what you think.